extras/courses.md

```diff
@@ -129,9 +129,11 @@ Courses | Duration | Effort
 :-- | :--: | :--:
 [Applied Cryptography](https://www.udacity.com/course/applied-cryptography--cs387)| 8 weeks | 6 hours/week
 [Cryptography](https://www.coursera.org/learn/cryptography/)| 7 weeks | 4-7 hours/week
-
-
+[Cryptography I](https://www.coursera.org/learn/crypto/)| 7 weeks | 4-7 hours/week
```

Comment:
We already have this course in the main curriculum. Could you remove this one (Cryptography I)?